libs/batch_environments/src/slurm_environment.cpp

```diff
@@ -279,92 +272,90 @@ namespace hpx { namespace util { namespace batch_environments
             qi::rule<std::string::iterator, std::string()> range_str;
             qi::rule<std::string::iterator, std::vector<std::string>()> range;
             qi::rule<std::string::iterator,
-                boost::fusion::vector<
-                    std::string,
-                    boost::optional<std::vector<std::vector<std::string> > >
-                >()> ranges;
+                boost::fusion::vector<std::string,
+                    boost::optional<std::vector<std::vector<std::string>>>>()>
+                ranges;
             qi::rule<std::string::iterator> hostlist;
             qi::rule<std::string::iterator> nodelist;
 
             // grammar definition
-            prefix %=
-                    +(qi::print - (qi::char_("[") | qi::char_(",")))
-                ;
+            prefix %= +(qi::print - (qi::char_("[") | qi::char_(",")));
 
-            range_str %=
-                    +(qi::print - (qi::char_("]") | qi::char_( ",") | qi::char_("-")))
-                ;
+            range_str %= +(
+                qi::print - (qi::char_("]") | qi::char_(",") | qi::char_("-")));
 
             range %= range_str >> *('-' >> range_str);
 
-            ranges %= prefix >> -(qi::lit("[") >> (range % ',') >> qi::lit("]"));
+            ranges %=
+                prefix >> -(qi::lit("[") >> (range % ',') >> qi::lit("]"));
 
-            hostlist =
-                (+ranges)[
-                    phoenix::bind(construct_nodelist(),
-                        phoenix::ref(nodes), qi::_1)
-                ];
+            hostlist = (+ranges)[phoenix::bind(
+                construct_nodelist(), phoenix::ref(nodes), qi::_1)];
 
             nodelist = hostlist % ',';
 
```

Comment:
Please do not reformat the Spirit parsers.

Reply:
Yep. My mistake.